dd-sdk-android-core/src/main/kotlin/com/datadog/android/api/net/RequestFactory.kt

```diff
@@ -67,5 +69,10 @@ fun interface RequestFactory {
          * Datadog tags query parameter name.
          */
         const val QUERY_PARAM_TAGS: String = "ddtags"
+
+        /**
+         * Datadog Idempotency key header, used for deduplication purposes.
```

Comment:
will it be used for any deduplication actually? I was under the impression that this is mostly for collecting the statistics right now

Reply:
Well yes maybe not for now, not sure should we change this definition ?

Reply:
I'm not even sure it will ever will. And anyway, if it will be, it will be used only for RUM events.